src/controllers/core/controller.js

```diff
@@ -109,9 +109,4 @@ class Controller
      *
      * @protected
      */
-    _updateMoney(amount)
```

Comment:
I wonder how you managed to remove this, but the result is at the very least, _amusing_. It happens to the best of us! 👍